style.css

```diff
@@ -0,0 +1,28 @@
+    #searchBar {
```

Comment:
I'm fine with this structure but if you are not going to use the `css` folder please delete it

Reply:
I'm using css for other properties like background color etc

Reply:
No what I mean is the files in this repo. It came with some starter files. If you don't want to use them delete them.
<img width="527" alt="Screenshot 2020-05-22 at 09 29 46" src="https://user-images.githubusercontent.com/13404719/82642882-e7c2cd00-9c0e-11ea-98c6-2027145f969e.png">